addons/tmf_loadouts/loadouts/chdkz.hpp

```diff
@@ -0,0 +1,510 @@
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch"
+	};
+
+	// These are put into the backpack
+	backpackItems[] = {};
+
+	// Unit traits, see https://community.bistudio.com/wiki/setUnitTrait
+	traits[] = {};
+
+	// This is executed after unit init is complete. argument: _this = _unit.
+	code = "";
+};
+
+class r : baseMan {
+	displayName = "Rifleman";
+	headgear[] = {
+		LIST_4("CUP_H_ChDKZ_Beanie"),
+		"CUP_H_RUS_SSH68_olive",
+		"CUP_H_RUS_SSH68_green",
+		LIST_2("")
+	};
+	goggles[] += {
+		"Default",
+		LIST_4(""),
+		"G_Balaclava_oli",
+		"G_Balaclava_blk",
+		"CUP_G_TK_RoundGlasses_blk",
+		"CUP_G_WristWatch"
+	};
+	primaryWeapon[] = {"CUP_arifle_AK74_Early"};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_545x39_AK_M"),
+		LIST_2("CUP_30Rnd_TE1_Yellow_Tracer_545x39_AK_M"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	primaryWeapon[] = {"CUP_arifle_AK74_GL_Early"};
+	backpack[] = {"CUP_B_SLA_Medicbag"};
+	magazines[] += {
+		LIST_8("CUP_1Rnd_HE_GP25_M"),
+		LIST_4("CUP_1Rnd_SMOKE_GP25_M")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	primaryWeapon[] = {"CUP_arifle_AKS74_Early"};
+	magazines[] = {
+		LIST_6("CUP_30Rnd_545x39_AK_M"),
+		LIST_2("CUP_30Rnd_TE1_Yellow_Tracer_545x39_AK_M"),
+		"CUP_HandGrenade_RGD5",
+		"SmokeShell"
+	};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	headgear[] = {
+		"CUP_H_C_Ushanka_01",
+		"CUP_H_C_Ushanka_02",
+		"CUP_H_C_Ushanka_03",
+		"CUP_H_C_Ushanka_04"
+	};
+	traits[] = {"medic"};
+	backpack[] = {"CUP_B_AlicePack_Bedroll"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	primaryWeapon[] = {"CUP_arifle_AKS74U"};
+	magazines[] = {
+		LIST_8("CUP_20Rnd_545x39_AKSU_M"),
+		LIST_2("CUP_HandGrenade_RGD5"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	items[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("CUP_1Rnd_SmokeRed_GP25_M")
+	};
+	linkedItems[] += {"Binocular","ItemGPS","ItemRadio"};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	headgear[] = {"CUP_H_ChDKZ_Cap"};
+	sidearmWeapon[] = {"CUP_hgun_Makarov"};
+	magazines[] += {
+		LIST_2("CUP_8Rnd_9x18_Makarov_M")
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"CUP_H_ChDKZ_Beret"};
+	goggles[] = {"G_Aviator"};
+	uniform[] = {"CUP_U_O_CHDKZ_Bardak"};
+};
+
+class fac : car {
+	displayName = "Forward Air Controller";
+	backpack[] = {"CUP_B_Kombat_Radio_Olive"};
+	backpackItems[] = { FAC_GEAR, "ACE_MapTools" };
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemRadio",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	backpack[] = {"CUP_B_ACRScout_m95"};
```

Comment:
Is this backpack intended? It's got a small capacity (1x 75rnd drum) and is a different camoflage scheme than the rest of the uniforms.

Reply:
Yep, it isn't too out of place with the current set of camos.